pom.xml

```diff
@@ -39,8 +39,8 @@
     </modules>
 
 	<properties>
-		<openMRSVersion>1.9.9</openMRSVersion>
-		<openMRSMinorVersion>1.9</openMRSMinorVersion>
+		<openMRSVersion>1.10.5</openMRSVersion>
+		<openMRSMinorVersion>1.10</openMRSMinorVersion>
 		<htmlwidgetsVersion>1.7.2</htmlwidgetsVersion>
```

Comment:
Hello @mseaton I had to do this because I couldn't resolve imports for the CareSetting class.
For the DrugOrder component of the CohortBuilder, I don't think I would be needing the CareSetting and hence I would revert back to 1.9.9 and remove imports for the CareSetting.

Also, I do not understand your second comment about locating all in the new reporting-1.10 module...